src/Umbraco.Infrastructure/Migrations/Install/DatabaseDataCreator.cs

```diff
@@ -19,6 +18,61 @@ namespace Umbraco.Cms.Infrastructure.Migrations.Install;
 /// </summary>
 internal class DatabaseDataCreator
 {
+
+    internal static readonly IEnumerable<LogViewerQueryDto> _defaultLogQueries = new LogViewerQueryDto[]
```

Comment:
Hmm, that's a good question, since it's an internal field it should be okay, I mean in general having fields accessed from outside a class is a bit dodgy since it breaks encapsulation, but I think it's okay in this case, I'll change it to an array 😄